Aardvark.xcodeproj/project.pbxproj

```diff
@@ -428,54 +464,130 @@
+				4551A2541BD9B56200F216D0 /* ARKLogMessage.m in Sources */,
+				4551A2641BD9B82200F216D0 /* NSURL+ARKAdditions.m in Sources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		EAD1442819E073FB0065A1FF /* Sources */ = {
+		4551A2331BD9B41000F216D0 /* Sources */ = {
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				EAAB38A419E2929C00161A54 /* ARKDefaultLogFormatterTests.m in Sources */,
-				EA46F7FA1ACB8448007FC415 /* ARKURLAdditionsTests.m in Sources */,
-				EA7353701A15C5E70060F54E /* ARKLogStoreTests.m in Sources */,
-				D04D47ED1AB8A0BA00A342E9 /* ARKDataArchiveTests.m in Sources */,
-				EA09D3C51A2E6D2F004C1125 /* ARKDefineTests.m in Sources */,
-				D04D48051AB9196B00A342E9 /* ARKFileHandleAdditionsTests.m in Sources */,
-				EAC8459C19F60B820000F165 /* ARKEmailBugReporterTests.m in Sources */,
-				EA82F7931A3F9D6B0043BA42 /* ARKLogTableViewControllerTests.m in Sources */,
-				EAD1445419E201C70065A1FF /* ARKLogDistributorTests.m in Sources */,
+				4551A2761BD9BA0F00F216D0 /* ARKDefaultLogFormatterTests.m in Sources */,
+				4551A27A1BD9BA0F00F216D0 /* ARKLogDistributorTests.m in Sources */,
+				4551A27C1BD9BA0F00F216D0 /* ARKLogTableViewControllerTests.m in Sources */,
+				4551A27D1BD9BA0F00F216D0 /* ARKURLAdditionsTests.m in Sources */,
+				4551A27B1BD9BA0F00F216D0 /* ARKLogStoreTests.m in Sources */,
+				4551A2771BD9BA0F00F216D0 /* ARKDefineTests.m in Sources */,
+				4551A2781BD9BA0F00F216D0 /* ARKEmailBugReporterTests.m in Sources */,
+				4551A2751BD9BA0F00F216D0 /* ARKDataArchiveTests.m in Sources */,
+				4551A2791BD9BA0F00F216D0 /* ARKFileHandleAdditionsTests.m in Sources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXSourcesBuildPhase section */
 
 /* Begin PBXTargetDependency section */
-		EAD1445619E203340065A1FF /* PBXTargetDependency */ = {
+		4551A23A1BD9B41000F216D0 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			target = EAD1442019E073FB0065A1FF /* Aardvark */;
-			targetProxy = EAD1445519E203340065A1FF /* PBXContainerItemProxy */;
+			target = 4551A22D1BD9B41000F216D0 /* Aardvark */;
+			targetProxy = 4551A2391BD9B41000F216D0 /* PBXContainerItemProxy */;
 		};
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		4551A2401BD9B41000F216D0 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 1;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEFINES_MODULE = YES;
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_TESTABILITY = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
+				INFOPLIST_FILE = Aardvark/Info.plist;
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
```

Comment:
if this is out any run time dependencies wouldn't be found. Below has some important goodies like `PRODUCT_BUNDLE_DENTIFIER` and `PRODUCT_NAME` that should stick around